pkg/validator/schema.go

```diff
@@ -250,7 +253,30 @@ func applySchemaCheck(conf *config.Configuration, checkID string, test schemaTes
 		}
 	} else if check.Target == config.TargetPod {
 		passes, issues, err = check.CheckPod(test.Resource.PodSpec)
+		if test.Resource.Kind == "CronJob" {
```

Comment:
Looking much better!

Can we just extract out a `getJSONSchemaPrefix` function to clean it up a little?